projects/storefrontlib/cms-structure/page/page-layout/page-layout.service.ts

```diff
@@ -19,11 +24,16 @@ export class PageLayoutService {
     private cms: CmsService,
     private config: LayoutConfig,
     private breakpointService: BreakpointService,
-    @Optional()
-    @Inject(PAGE_LAYOUT_HANDLER)
-    private handlers: PageLayoutHandler[]
+    // TODO: need doc this in 5.0
+    //@Optional()
+    //@Inject(PAGE_LAYOUT_HANDLER)
+    //private handlers: PageLayoutHandler[],
+    private unifiedInjector: UnifiedInjector
   ) {}
 
+  private handlers$: Observable<PageLayoutHandler[]> =
+    this.unifiedInjector.get(PAGE_LAYOUT_HANDLER);
+
```

Comment:
What's the purpose of this logic?

Reply:
get all `PageLayoutHandler` even they are injected in lazy loaded modules.